modules/misc/doc/content/source/postprocessors/ThermocoupleSensorPostprocessor.md

```diff
@@ -0,0 +1,38 @@
+# ThermocoupleSensorPostprocessor
+
+!syntax description /Postprocessors/ThermocoupleSensorPostprocessor
+
+## Description
+
+`ThermocoupleSensorPostprocessor` object implements a thermocouple sensor that takes in the calculated input signal from moose and outputs a realistic and sensor-mediated output that accounts for efficiency, drift, delay and noise of the sensor in question. Different types of thermocouples are included. This is a child class of `GeneralSensorPostprocessor`.
+
+From the `GeneralSensorPostprocessor`,
```

Comment:
do you mean the thermo couple?

Reply:
no, it's the equation from the GeneralSensorPostprocessor.